schedule/yast/qam-yast_self_update_libyui-rest.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: qam-yast_self_update_libyui-rest
```

Comment:
let's use a name that doesn't contain libyui in file name or here, later we will figure out nomenclature.